pkgs/development/python-modules/pypdftk/default.nix

```diff
@@ -0,0 +1,36 @@
+{ stdenv
+, buildPythonPackage
+, fetchPypi
+#, glibcLocales
+, python
+, pdftk
+, isPy3k
+}:
+
+buildPythonPackage rec {
+  pname = "pypdftk";
+  version = "0.4";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "0kl1kyqi5d8lwzlbv8430qvzz4q2jg8nv5sakd1i1qmnkn61g5n5";
+  };
+
+  #LC_ALL = "en_US.UTF-8";
+  #buildInputs = [ glibcLocales ];
+
+  checkPhase = ''
+    ${python.interpreter} test.py
+  '';
+
+  # Tests broken on Python 3.x
+  doCheck = !(isPy3k);
```

Comment:
Is there an issue upstream about this? Does pypdftk itself work with python 3?

Reply:
I wanted to evaluate this for something today and used this expression as a base. I got pypdftk and the tests working in python 3, so I think these lines can be cut. There are a few leaps to getting the tests working from here, but I'm going to make a general comment on the issue with what I found so that it doesn't get hidden if/when this is marked resolved.